main.py

```diff
@@ -107,27 +107,51 @@ async def on_message(message: Message):
         await command_stop()
 
 
-# Take a filename as string and return it formatted nicely
-def format_filename(filename: str):
+# Format a song as text nicely using artist/title tags if available
+# local_filepath is the actual path on disc
+# filename is the filename on Discord
+# artist_fallback is the fallback artist value (no fallback if not passed)
```

Comment:
We can probably remove this now that the docstring contains the same content.

Reply:
Ah yeah, I left those in there by accident